R/config.R

```diff
@@ -0,0 +1,100 @@
+#' Fetch the config from an external resource
+#'
+#' This step is the first part of the modeling pipeline. It looks to Azure Blob
+#' and downloads the Rt model run's config to the local config (if
+#' `blob_storage_container` is specified), reads the config in from the
+#' filesystem, and validates that it matches expectations. If any of these steps
+#' fails, the pipeline fails with an informative error message. Note, however,
+#' that a failure in this initial step suggests that something fundamental is
+#' misspecified and the logs will likely not be preserved in a Blob Container if
+#' running in Azure.
+#'
+#' The validation relies on `inst/data/config_schema.json` for validation. This
+#' file is in `json-schema` notation and generated programatically via
+#' https://www.jsonschema.net/.
+#'
```

Comment:
General comment, I think these are all character strings but I find it helpful when reading documentation when the type is explicitly specified.